compiler/src/dotty/tools/dotc/core/Definitions.scala

```diff
@@ -1006,6 +1006,7 @@ class Definitions {
   @tu lazy val ProvisionalSuperClassAnnot: ClassSymbol = requiredClass("scala.annotation.internal.ProvisionalSuperClass")
   @tu lazy val DeprecatedAnnot: ClassSymbol = requiredClass("scala.deprecated")
   @tu lazy val DeprecatedOverridingAnnot: ClassSymbol = requiredClass("scala.deprecatedOverriding")
+  @tu lazy val deprecatedInheritance: ClassSymbol = requiredClass("scala.deprecatedInheritance")
```

Comment:
the convention here is leading uppercase and suffix `Annot`. (I understand the urge to disregard convention.)

Reply:
I was about to fix it right now 😄